lib/source-destination/http.ts

```diff
@@ -56,7 +56,13 @@ export class Http extends SourceDestination {
 
 	private async getInfo() {
 		try {
-			const response = await axios({ method: 'head', url: this.url });
+			// We use GET instead of HEAD as some servers will respond with a 403 to HEAD requests
```

Comment:
It might be worth trying a `HEAD` and then falling back to a `GET` because I think it's possible to optimize specifically for `HEAD` right? (I think we do that for some stuff ourselves?)